components/ws-manager-mk2/service/manager.go

```diff
@@ -284,6 +284,15 @@ func (wsm *WorkspaceManagerServer) StartWorkspace(ctx context.Context, req *wsma
 	}
 	controllerutil.AddFinalizer(&ws, workspacev1.GitpodFinalizerName)
 
+	exists, err := wsm.workspaceExists(ctx, req.Metadata.MetaId)
```

Comment:
from https://linear.app/gitpod/issue/WKS-250#comment-3fda647f, I think we should still allow the creation of a new instance if the older instance(s) are in a `stopped` state. Otherwise this will block the restart of a workspace until the `stopped` workspace clears (which we've seen can take some time)